README.md

```diff
@@ -47,20 +51,15 @@ Terraform state outputs.
 | a_string | foo | string
 ```
 
-## More options
+</details>
 
-```
-terraputs -h
-Usage of terraputs:
-  -heading string
-        Optional; the heading text for use in the printed markdown (default "Outputs")
-  -state string
-        Optional; the state JSON output by 'terraform show -json', read from stdin if omitted
-  -state-file string
-        Optional; the path to a local file containing 'terraform show -json' output
-```
+<details>
 
-## Example output table formatted by GitHub
+<summary>Markdown output rendered via GitHub-flavored markdown</summary>
```

Comment:
I took the initiative to improve the `README` formatting a bit. Its GitHub-rendered result can be viewed here: https://github.com/mdb/terraputs/tree/html-output#results-examples